client/landing/stepper/utils/flow-redirect-handler.ts

```diff
@@ -1,12 +1,12 @@
 import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
 
 // Flows to redirect
-const redirectRoutes = [
-	{ from: '/setup/blog/', to: '/start:lang?' },
-	{ from: '/setup/free/', to: '/start/free:lang?' },
-	{ from: '/setup/link-in-bio/', to: '/start:lang?' },
-	{ from: '/setup/videopress/', to: '/start:lang?' },
-	{ from: '/setup/sensei/', to: ':lang?/plugins/sensei-pro/' },
+export const REMOVED_TAILORED_FLOWS = [
+	{ flow: 'blog', to: '/start:lang?' },
+	{ flow: 'free', to: '/start/free:lang?' },
+	{ flow: 'link-in-bio', to: '/start:lang?' },
+	{ flow: 'videopress', to: '/start:lang?' },
+	{ flow: 'sensei', to: ':lang?/plugins/sensei-pro/' },
```

Comment:
We can use the Flow constants here.

Reply:
https://github.com/Automattic/wp-calypso/pull/95669/commits/b4cb7bf7517a0fb912438e4df1a1a32b9babba01